api/server/sdk/volume_ops.go

```diff
@@ -709,6 +709,25 @@ func (s *VolumeServer) CapacityUsage(
 	return resp, nil
 }
 
+/*
+func (s *VolumeServer) VolumeUsageByNode(
+	ctx context.Context,
+	req *api.SdkVolumeUsageByNodeRequest,
+) (*api.SdkVolumeUsageByNodeResponse, error) {
+
+	resp, err := s.driver(ctx).VolumeUsageByNode(req.GetNodeId())
```

Comment:
should we check permission for each volume returned by the API ? 
I think simply adding the following should be enough once you have the response:

@ggriffiths  can you confirm if this is all we need ?

```
// Get spec. This also checks if the parend id exists.
	// This will also check for Ownership_Read access.
	parentVol, err := s.Inspect(ctx, &api.SdkVolumeInspectRequest{
		VolumeId: volumeID,
	})
	if err != nil {
	    // user does not have permission ? 	
	}

```

Reply:
This was commented code that i meant to remove. Sorry missed removing.  The security context comes from volume annotations? and since there are multiple volumes and it's really for the node, moved the api to node service.